core/main/src/broker/endpoint_broker.rs

```diff
@@ -404,6 +414,25 @@ impl EndpointBrokerState {
         }
     }
 
+    fn handle_static_request(
+        &self,
+        rpc_request: RpcRequest,
+        extn_message: Option<ExtnMessage>,
+        rule: Rule,
+        callback: BrokerCallback,
+    ) {
+        if rule.transform.response.is_some() {
```

Comment:
`transform.response` is required for any rule that has a static alias, so I'm not really sure we need this check.

Reply:
No point in having static rule with response, that's why there was a check.  
Removed the check, static rule without response will return 'result: ""'